From 1a1b31b65aaaeb1ce1ac676b8dc28966c9458544 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Fri, 4 Mar 2016 16:53:55 +0100
Subject: [PATCH] usb: Add native DSD support for Amanero Combo384

This patch adds native DSD support for the Amanero Combo384.
It uses the default VID/PID (16d0:071a) and requires firmware 1099rc1.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 5b03296..52acb82 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1286,6 +1286,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 
 	/* XMOS based USB DACs */
 	switch (chip->usb_id) {
+	case USB_ID(0x16d0, 0x071a): /* Amanero Combo384 */
 	case USB_ID(0x20b1, 0x3008): /* iFi Audio micro/nano iDSD */
 	case USB_ID(0x20b1, 0x2008): /* Matrix Audio X-Sabre */
 	case USB_ID(0x20b1, 0x300a): /* Matrix Audio Mini-i Pro */
-- 
2.5.0

